tools/clang/include/clang/Basic/LangOptions.h

```diff
@@ -39,8 +40,10 @@ class LangOptionsBase {
 
 #else
 
-#define LANGOPT(Name, Bits, Default, Description) static const unsigned Name = Default;
-#define LANGOPT_BOOL(Name, Default, Description) static const bool Name = static_cast<bool>( Default );
+#define LANGOPT(Name, Bits, Default, Description)                              \
+  static const unsigned Name = Default;
+#define LANGOPT_BOOL(Name, Default, Description)                               \
+  static const bool Name = static_cast<bool>(Default);
```

Comment:
How were all these unrelated formatting changes necessary for LangOptions.h?
Could this have been left-over from trying to accommodate clang-format checker errors after switching base branch on the previous PR?  Maybe it wouldn't have complained about these in this branch had you only added the original modification?

Reply:
Yes, I think the most likely explanation or cause was the base branch switch.